src/hooks/helpers.ts

```diff
@@ -0,0 +1,51 @@
+import { useReducer } from "react";
```

Comment:
can rename the file ? `useEthereumTransactionState`

Reply:
Changed it